mods/cnc/rules/defaults.yaml

```diff
@@ -767,6 +781,8 @@
 	Interactable:
 	CombatDebugOverlay:
 	AppearsOnRadar:
+	OwnerLostAction:
+		Action: ChangeOwner
```

Comment:
I like the idea of keeping walls around after defeat, but this is currently mixing refactoring changes with gameplay changes.  Can you please keep these as Kill in the first commit, and then introduce a new commit that changes them to ChangeOwner, like you did for the tech structures?